tests/functional/scripts/pyi_helloworld.py

```diff
@@ -0,0 +1,11 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+print('Hello Python!')
```

Comment:
1. The corresponding change to the test-suite is missing, isn't it?
2. For such simple test-scripts please use `pyi_builder.test_source()`, which avoids additional files.